python/graphscope/config.py

```diff
@@ -80,4 +80,9 @@ class GSConfig(object):
     # GIE engine params
     engine_params = None
 
+    # GIE instance will be created automatically when a property graph loaded.
+    # Otherwise, you should create a GIE instance manually by `sess.gremlin` if
+    # `auto_create_interactive_engine` is False
+    auto_create_interactive_engine = True
```

Comment:
naming: `initializing_interactive_engine` would be better.

But current option name is fine as well.